.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Thanks for contributing!
+-->
```

Comment:
```suggestion
<!-- Thanks for contributing! -->
```

/me puts on his curmudgeon hat

If this line has to stay, making it a single line will at least make it easier to delete before opening the PR.